blog/2022-08-04-Building-interactive-data-applications-in-NeuroBlu-with-Plotly-Dash.md

```diff
@@ -0,0 +1,89 @@
+
+Embedding Plotly Dash apps allows for interactive visualizations in the NeuroBlu Research platform that can be used to view key information that helps decide the right cohort for research study.
+
+## Insights Explorer on NeuroBlu Research
+
+The Insights Explorer feature allows you to get quick Insights for your research study that might help you with any of the following:
+
+1. Deciding if the datasets within NeuroBlu are right for your research study.
+2. Deciding the right cohort inclusion and exclusion criteria for your research study.
+3. Coming up with the research protocol for your research study.
+
+Below is a highlight of apps developed on Plotly Dash Enterprise and embedded in NeuroBlu Research.
+
+**Exploring a disorder group**
+
+Under the “View Disorder Trends” tab, the user can select a dataset and disorder group to display the following information on that group:
+
+- Number of patients diagnosed with disorder
+- Demographic characteristics
+    - Gender distribution
+    - Age distribution
+    - Race distribution
+- Epidemiology and visits data
+    - Incidence
+    - Visit distribution
+    - Outcome distribution
+    - Observation distribution
+    - First captured CGI-S
+- Disorder trends
+    - Common comorbidities
+    - Most prescribed drugs
+    - Number of 1 year follow ups
+    - Duration of hospitalizations distribution
+    - Number of hospitalizations distribution
+    - Top stressors
+
+**Comparing cohorts by drugs**
+
+Under the “Compare Drugs” tab, the user can select a database, a category mapper diagnosis, and 2 different drugs to display the following information: 
+
+- Number of patients in drug group 1
+- Number of patients in drug group 2
+- Patient group characteristics
+    - Gender distribution
+    - Age distribution
+    - Race distribution
+    - Sankey plots highlighting the following across drug groups
+        - Treatment switches
+        - Hospitalization results
+    
+
+## Using Plotly
+
+- Learned how to leverage python in our own library
+- Dealt with constraints with features and customizations
+- Resource-intensive environment
+- Some out-of-the-box components are less sophisticated than alternatives, but more extensible
+
+## Next Steps
+
+1. Explore the [Dashboard Engine](https://plotly.com/dash/dashboard-engine/) released in December 2021
+2. Continue leveraging the multi-page applications that we are developing
+3. Test database connectors to accelerate computational speed
+4. Look at open-source integration with Haskell and Elm frameworks
```

Comment:
This point is a bit weird for few reasons:

1. we already do integrate with dash
2. I'm not sure how Haskell fits into this scheme. There is no need for backed integration between dash and our system I can't think of how would that work.
3. There are no open-source integrations and can't be due to licensing policy of Dash. They offer React.js component under proprietary license. We integrate with them via our own custom React wrapper app which is further wrapped into WebComponent with which we integrate via DOM from Elm. But that's is probably not even worth mentioning as it would raise more question than answer.

Reply:
@turboMaCk I think this blogpost was written by one of our summer interns, who I think is no longer around to make ammendments. Unless you have suggestions to make the formulations more precise, I don't think there's anyone around to push this blogpost over the merge-line :smile:

Reply:
what if we would just remove 4th point then?